src/Common/src/Interop/FreeBSD/Interop.Libraries.cs

```diff
@@ -5,6 +5,12 @@ internal static partial class Interop
 {
     private static partial class Libraries
     {
-        internal const string LibRt = "librt"; // POSIX Realtime Extensions library
+        /// <summary>
+        /// We aren't OS X so don't have an INODE64 suffix to entry points
+        /// </summary>
+        internal const string INODE64SUFFIX = "";
+
+        internal const string LibRt = "librt";  // POSIX Realtime Extensions library
+        internal const string libc = "libc";    // C runtime
```

Comment:
I can't include the Unix Libraries.cs file since that's included in OS X and Linux; I need to declare the INODE64SUFFIX constant for each platform (ie the Libraries.cs file) since there isn't a shared file between BSD and Linux that doesn't also include OS X. Due to this constant and the niceness of code re-use by this refactor, it made it necessary to include libc in each platform's Libraries.cs. It seemed like a small price to pay for all the goodness we get from code re-use; however, I'm definitely open to cleaner solutions :)